test/JDBC/expected/test_db_collation-vu-verify.out

```diff
@@ -0,0 +1,1800 @@
+~~END~~
+
+
+SELECT name, collation_name FROM SYS.TYPES WHERE collation_name IS NOT NULL ORDER BY name + COALESCE(collation_name, '');
+GO
+~~START~~
+varchar#!#varchar
+char#!#bbf_unicode_cp1_ci_as
+ct#!#bbf_unicode_cp1_ci_as
+nchar#!#bbf_unicode_cp1_ci_as
+ntext#!#bbf_unicode_cp1_ci_as
+nvarchar#!#bbf_unicode_cp1_ci_as
+sql_variant#!#bbf_unicode_cp1_ci_as
+sysname#!#bbf_unicode_cp1_ci_as
+text#!#bbf_unicode_cp1_ci_as
+varchar#!#bbf_unicode_cp1_ci_as
+~~END~~
+
+
+CREATE TYPE ct FROM NVARCHAR(11);
+GO
+~~ERROR (Code: 219)~~
+
+~~ERROR (Message: type "ct" already exists)~~
+
+
+SELECT name, collation_name FROM SYS.TYPES WHERE name = 'ct';
+GO
+~~START~~
+varchar#!#varchar
+ct#!#bbf_unicode_cp1_ci_as
+~~END~~
+
+
+-- Case 4.1: Computed columns
+USE test_db_collation_vu_prepare_db121;
+GO
+
+SELECT * FROM test_db_collation_vu_prepare_db121_computed_columns ORDER BY a;
+GO
+~~START~~
+nvarchar#!#nvarchar
+Año Nuevo#!#oveuN oñA
+café#!#éfac
+chaptéR#!#Rétpahc
+película#!#alucílep
+~~END~~
+
+
+SELECT * FROM test_db_collation_vu_prepare_db121_computed_columns ORDER BY b;
+GO
+~~START~~
+nvarchar#!#nvarchar
+película#!#alucílep
+café#!#éfac
+Año Nuevo#!#oveuN oñA
+chaptéR#!#Rétpahc
+~~END~~
+
+
+-- Case 4.2: Temp Tables
+CREATE TABLE #test_db_collation_vu_prepare_db121_temp_t1(a nvarchar(11), b nvarchar(11), c AS sys.reverse(b) NOT NULL)
+GO
+
+INSERT INTO #test_db_collation_vu_prepare_db121_temp_t1 VALUES
+('café', 'jalapeño'),
+('résumé', 'naïve'),
+('Piñata', 'Año Nuevo'),
+('TELÉFONO', 'película'),
+('árbol', 'canapé'),
+('chaptéR', 'TEññiȘ');
+GO
+~~ROW COUNT: 6~~
+
+
+SELECT * FROM #test_db_collation_vu_prepare_db121_temp_t1 WHERE c = 'evian'; -- reverse of naive
+GO
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar
+résumé#!#naïve#!#evïan
+~~END~~
+
+
+SELECT * FROM #test_db_collation_vu_prepare_db121_temp_t1 ORDER BY c;
+GO
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar
+TELÉFONO#!#película#!#alucílep
+árbol#!#canapé#!#épanac
+résumé#!#naïve#!#evïan
+café#!#jalapeño#!#oñepalaj
+Piñata#!#Año Nuevo#!#oveuN oñA
+chaptéR#!#TEññiȘ#!#ȘiññET
+~~END~~
+
+
+CREATE TABLE #test_db_collation_vu_prepare_db121_temp_t2(a nvarchar(11), b nvarchar(11), c AS sys.reverse(b) NOT NULL)
+GO
+
+INSERT INTO #test_db_collation_vu_prepare_db121_temp_t2 VALUES
+('café', 'jalapeño'),
+('résumé', 'naïve'),
+('Piñata', 'Año Nuevo'),
+('TELÉFONO', 'película'),
+('árbol', 'canapé'),
+('chaptéR', 'TEññiȘ');
+GO
+~~ROW COUNT: 6~~
+
+
+SELECT t1.a, t2.a FROM #test_db_collation_vu_prepare_db121_temp_t1 t1 INNER JOIN #test_db_collation_vu_prepare_db121_temp_t2 t2 on t1.a = t2.a ORDER BY t1.a;
+GO
+~~START~~
+nvarchar#!#nvarchar
+árbol#!#árbol
+café#!#café
+chaptéR#!#chaptéR
+Piñata#!#Piñata
+résumé#!#résumé
+TELÉFONO#!#TELÉFONO
+~~END~~
+
+
+-- Drop the temp tables
+DROP TABLE #test_db_collation_vu_prepare_db121_temp_t1;
+GO
+
+DROP TABLE #test_db_collation_vu_prepare_db121_temp_t2;
+GO
+
+-- Case 4.3: Table Returning Functions
+SELECT * FROM test_db_collation_vu_prepare_db121_f1();
```

Comment:
Table returning function internally creates table, lets verify the column metadata of the same

Reply:
Updated by calling sp_describe_first_result_set